recipe/meta.yaml

```
@@ -13,22 +15,32 @@ source:
 
 build:
   number: 1005
```

If we are one version after conda-forge, it's a new version therefore we must have a build number to 0

```suggestion
  number: 0
```

---

Duh!
Hmm, how do I automate a check?  🤔

---

A new version => reset build number to 0.